src/gp_engine.cpp

```diff
@@ -17,11 +15,8 @@ GPEngine::GPEngine(SitePattern site_pattern, size_t node_count,
     : site_pattern_(std::move(site_pattern)),
       rescaling_threshold_(rescaling_threshold),
       log_rescaling_threshold_(log(rescaling_threshold)),
-      plv_count_per_node_(plv_count_per_node),
-      mmap_file_path_(mmap_file_path),
-      mmapped_master_plv_(mmap_file_path_,
-                          (node_count + node_padding_) * plv_count_per_node_ *
-                              size_t(resizing_factor_) * site_pattern_.PatternCount()),
+      plv_handler_(mmap_file_path, size_t(0), site_pattern_.PatternCount(),
```

Comment:
Is this a cast of 0 to `size_t`? Is it needed?

Reply:
Nope, sorry.  That was from a templating issue I was having.  Fixed.